index.js

```diff
@@ -1,10 +1,19 @@
-import {NativeModules, DeviceEventEmitter} from 'react-native';
+import {NativeModules, DeviceEventEmitter, Platform, PushNotificationIOS} from 'react-native';
```

Comment:
since we include PushNotificationIOS here, can we bridge request permission and check permission on it?